code/modules/admin/verbs/admincryo.dm

```diff
@@ -0,0 +1,57 @@
+/client/proc/cmd_admin_cryo(mob/living/M as mob in GLOB.living_mob_list_)
+	set category = "Special Verbs"
+
+	set name = "Admin Cryo"
+	if(!check_rights(R_MOD))
+		src << "Only moderators may use this command."
+		return
+	if(!istype(M))
+		alert("Cannot cryo a ghost")
+		return
+
+	var/confirm = alert(src, "You will be removing [M] from the round, are you sure?", "Message", "Yes", "No")
+	if(confirm != "Yes")
+		return
+	if (usr.client)
+		if(usr.client.holder)
+
+			for(var/obj/item/card/id/Z in M)
+				qdel(Z)
+			for(var/obj/item/device/pda/Y in M)
+				qdel(Y)
+			for(var/obj/item/clothing/under/X in M)
+				qdel(X)
+			for(var/obj/item/organ/V in M)
+				qdel(V)
```

Comment:
what is the point of this if the mob is getting qdel'd in like 10 lines from here